### PR TITLE
Waveshare ESP32-S3-Touch-LCD-4.3 board

### PR DESCRIPTION
**Pull request by lboue**
Add support for [Waveshare ESP32-S3-Touch-LCD-4.3](https://www.waveshare.com/wiki/ESP32-S3-Touch-LCD-4.3) board

### CODE REVIEW

**Comment by lboue** [comment]
Serial Boot log
```
============ Before Setup End ============
Squareline WiFi clock example start
[   917][W][STA.cpp:537] disconnect(): STA already disconnected.
Initialize panel device
D (430) ESP_Panel: Panel init start
D (431) ESP_Panel: Use LCD
D (431) ESP_Panel: Use RGB bus
D (431) ESP_Panel: Create LCD bus
D (433) ESP_Panel: Create LCD device
D (436) ESP_Panel: Use touch
D (439) ESP_Panel: Use I2C bus
D (442) ESP_PanelHost: Add host I2C[0]
D (445) ESP_Panel: Create touch bus
D (448) ESP_Panel: Create touch device
D (452) ESP_Panel: Use IO expander
D (455) ESP_Panel: Initialize host
D (458) ESP_PanelHost: Initialize host I2C[0]
D (462) ESP_Panel: Panel init end
D (465) ESP_PanelHost: Destroyed
D (468) ESP_Panel: Panel begin start
D (471) ESP_Panel: Begin IO expander
D (475) ESP_Panel: Begin LCD
D (500) ST7262_CPP: LCD panel @0x3fca8e6c created
D (500) ESP_PanelLcd: Begin start
D (501) ESP_PanelLcd: Begin end
D (501) ESP_Panel: Begin touch
D (502) ESP_PanelBus_I2C: Panel IO @0x3fcb0e38 created
D (507) ESP_PanelTouch: Disable interruption
D (511) GT911_CPP: Use default GT911 driver data(address: 0x5d)
I (517) GT911: LCD touch panel create success, version: 1.1.1
W (522) GT911: Unable to initialize the I2C address
I (527) GT911: TouchPad_ID:0x39,0x31,0x31
I (531) GT911: TouchPad_Config_Version:67
D (534) ESP_Panel: Panel begin end
Initialize LVGL
[  1041][D][lvgl_port_v8.cpp:610] lvgl_port_init(): [lvgl_port] Initialize LVGL display driver
[  1051][D][lvgl_port_v8.cpp:447] display_init(): [lvgl_port] Malloc memory for LVGL buffer
[  1059][D][lvgl_port_v8.cpp:454] display_init(): [lvgl_port] Buffer[0] address: 0x3fcb10c4, size: 32000
[  1069][D][lvgl_port_v8.cpp:454] display_init(): [lvgl_port] Buffer[1] address: 0x3fcb8dd4, size: 32000
[  1078][D][lvgl_port_v8.cpp:485] display_init(): [lvgl_port] Register display driver to LVGL
[  1089][D][lvgl_port_v8.cpp:404] update_callback(): [lvgl_port] Update display rotation to 0
[  1098][D][lvgl_port_v8.cpp:405] update_callback(): [lvgl_port] Current mirror x: 0, mirror y: 0, swap xy: 0
[  1108][D][lvgl_port_v8.cpp:623] lvgl_port_init(): [lvgl_port] Initialize LVGL input driver
[  1116][D][lvgl_port_v8.cpp:537] indev_init(): [lvgl_port] Register input driver to LVGL
[  1124][D][lvgl_port_v8.cpp:639] lvgl_port_init(): [lvgl_port] Create mutex for LVGL
[  1132][D][lvgl_port_v8.cpp:643] lvgl_port_init(): [lvgl_port] Create LVGL task
[  1139][D][lvgl_port_v8.cpp:568] lvgl_port_task(): [lvgl_port] Starting LVGL task
Create UI
Wifi connecting...
```

**Comment by lboue** [comment]
@Lzw655 Could you also add this change in the release v0.1.6?

File: src/board/ESP_PanelBoard.h
Comment (Lzw655) [inline src/board/ESP_PanelBoard.h:1]: Please also add macros `BOARD_ELECROW_CROWPANEL_7_0` and `BOARD_WAVESHARE_ESP32_S3_Touch_LCD_4_3` here. And increase the number of macros `ESP_PANEL_BOARD_SUPPORTED_FILE_VERSION_MINOR` in the file *ESP_Panel_Board_Supported.h* and `ESP_PANEL_BOARD_SUPPORTED_VERSION_MINOR` in the file *src/ESP_PanelVersions.h*. Thank you!

https://github.com/esp-arduino-libs/ESP32_Display_Panel/blob/13bbe765b2b862b07f72e0fd77988babfdbd833a/src/board/ESP_PanelBoard.h#L11-L21

**Review by Lzw655 — APPROVED** [review]
Others LGTM. Thanks for your contributions! 👍

File: src/board/ESP_PanelBoard.h
Comment (lboue) [reply src/board/ESP_PanelBoard.h:1]: Done

**Pull request merged**